data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,114 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		cost 35100000
+		shields 26700
+		hull 19600
+		"required crew" 83
+		bunks 91
+		mass 870
+		drag 9.8
+		"outfit space" 887
+		"cargo space" 27
+		"weapon capacity" 364
+		"engine capacity" 150
+		"heat dissipation" 0.675
+		# Expensive features inherited from Ladybug-era technology, with some fuel (and improved heat dissipation):
+		"tactical scan power" 30
+		ramscoop 1.5
+		"fuel capacity" 800
+		weapon
+			"shield damage" 5000
+			"hull damage" 2500
+			# High hit force and wide radius for a spectacle if the leader dies.
+			# This should be adjusted downward if the ship is reused more widely.
+			"blast radius" 800
+			"hit force" 8000
+	outfits
+		"Thermal Repeater Rifle" 25
+		"Thermal Repeater Turret" 4
+		"Thunderhead Launcher" 6
+		"Thunderhead Missile" 320
+		"Thunderhead Storage Array" 4
+		"Warder Anti-Missile" 2
+		
+		"Liquid Sodium Cooler"
+		"Jump Drive"
+		"Quantum Keystone"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Outfits Expansion"
+		
+		"White Sun Reactor"
```

Comment:
At the outset, the Unfettered raids on Wanderer space are pretty limited in scope. It's generally not that many ships and they're not particularly aggressive. They seem to mostly be in the food stage. Perhaps they shouldn't have the current largest reactor the Wanderers are making as well as a full complement of ammunition at this stage?

Reply:
This is just what one deep river carries actually for the weapon
As for the generator its just on all the strongwinds so not hard to get

This leader has the influence needed to get those by far; he just went there with 2 water bugs and collected it

Reply:
Because the Deep River's not going to use any of its ammo?

Looking at the nearest Wanderer systems at this point in the game, the Unfettered are not very successful at disabling the Wanderer ships. It's not even really close.

Reply:
Some ammo will be left on the other ships and some Uhai will bring back ammo with the food

Reply:
Maybe he should have less than 320?
Not so much as to make him weak but enough to show these are in limited supply, say 142*2 = 284, two shield beetles full of them?

Reply:
Now using Unfettered weaponry so this should be good